README.md

```diff
@@ -106,6 +107,19 @@ Example Playbooks
           - nginx_status_url: http://example2.com:1234/nginx_status/
             tags:
               - instance:bar
+        logs:
```

Comment:
can you mention here (and on the `log_enabled` line above) that this is an Agent 6 feature only ? (I'm thinking this might be a bit confusing for agent 5 users otherwise)

Reply:
@olivielpeau done